terraform/cluster/aws/main.tf

```diff
@@ -155,6 +155,8 @@ resource "aws_launch_template" "cluster" {
     }
   }
 
+  instance_type = var.node_type
```

Comment:
will it cause issue if multiple node type specified
```
Node instance type. You can also pass a comma separated list of instance types
```

Reply:
node_type is type of string, it only accepts one instance type.

![image](https://github.com/convox/convox/assets/8239709/47f1e9cf-d69b-42e6-8e2c-9b5a5321b545)

I can update the documentation to say that is only one instance type.

Reply:
See https://github.com/convox/convox/blob/55f4401b13aff7b23901c9294dbb5684060071a6/terraform/cluster/aws/variables.tf#L85